telicent_lib/config/configurator.py

```diff
@@ -143,7 +145,8 @@ def __handle_errors__(self, message: str, on_error: OnError):
         :param on_error: On Error behaviour
         """
         if on_error is OnError.EXIT:
-            print(message)
+            logger.error(message)
```

Comment:
best using logger.critical for this I think, so it's easier to surface.

Reply:
Good spot, I'll make the change. Thanks